xarm_gripper/urdf/xarm_gripper.urdf.xacro

```diff
@@ -246,7 +246,7 @@
         <pose>0 -0.015 0.015 0 0 0</pose>
         <axis>
           <xyz>1 0 0</xyz>
-          <use_parent_model_frame>true</use_parent_model_frame>
+          <use_parent_model_frame>false</use_parent_model_frame>

```

Comment:
I believe `use_parent_ model_frame` is deprecated from sdf version 1.7 and onwards.
http://sdformat.org/spec?elem=joint&ver=1.7

Reply:
Leaving this here is ok, since by default the joint will reference to joint frame instead of parent frame